CaloMC/fcl/common.fcl

```diff
@@ -0,0 +1,6 @@
+BEGIN_PROLOG
```

Comment:
can we move this `common.fcl` in the `CaloReco` area? Some of these numbers are used also by modules in the `DAQ` area, so I think it would be cleaner to have them not in a MC package

Reply:
TBH, the only parameter I care in the Trigger reco is the DigiSampling, which is a constant that will `never` change . The calorimeter readout controller board design is frozen (except that problem with the VTRx...), so the ADC sampling frequency maybe can be defined as a constant in the modules where we need it?

Reply:
If we want to use the code for Mu2e II, then we'll change the digiSampling, so it's always a good idea to have it as a parameter. Is there a place to store global calo parameters until we have them in proditions?

Reply:
can we add it in the `Mu2eG4/test/conditions_01.txt` file?